scholia/app/templates/taxon_metabolome.sparql

```diff
@@ -14,7 +14,7 @@ WITH {
     UNION
```

Comment:
Would remove completely

Reply:
Thanks ! Took into account in [359f2e8](https://github.com/WDscholia/scholia/pull/2334/commits/359f2e8130a8bda02860e757d22c6dc603b12e42)